core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListManager.java

```diff
@@ -29,18 +30,14 @@
 import java.util.*;
 import java.util.concurrent.locks.ReentrantLock;
 
-public class SimplifiedMasternodeListManager extends AbstractManager {
+public class SimplifiedMasternodeListManager extends AbstractManager implements QuorumStateManager {
```

Comment:
So this class is only for the DIP-04 method? I'm wondering if a comment on this can be useful here

Reply:
DIP4 is handled by `QuorumState` and DIP24 by `QuorumRotationState` - This class will make sure that the appropriate DIP's are being handled based on which is active.

Reply:
Comments added.